### DISCUSSION TITLE
Auto Tag with current context when Locked

### DISCUSSION THREAD

**Discussion by saphirahopeseeker** [standardnotes/forum#2553] [General]
### Reproduction steps:

1. Open the web or desktop app
2. Make sure you have a Tag X.Y 
2. Click on Tag X (or Tag X.Y) from the tags pane
2. Delete Tag X (or Tag X.Y) from the note itself
3. Click **Options** > **Lock**

### Expected behavior: 
The note is locked and no changes are made to it.

### Actual behavior: 
Tag X (or Tag X.Y) is re-added and then the note is locked.

**Comment by JaspalSuri** [comment]
Hi @standardorcawolf, thank you for reporting this issue! I was able to reproduce it and will create a report for the dev team to review.
